src/main/java/tech/jhipster/lite/generator/server/springboot/thymeleaf/template/domain/ThymeleafTemplateModuleFactory.java

```diff
@@ -132,7 +132,7 @@ public JHipsterModule buildTailwindcssModule(JHipsterModuleProperties properties
       .packageJson()
         .addDevDependency(packageName("tailwindcss"), COMMON)
         .addScript(scriptKey("watch:html"), scriptCommand("onchange 'src/main/resources/templates/**/*.html' -- npm-run-all --serial build:css build:html"))
-        .addScript(scriptKey("watch:serve"), scriptCommand("browser-sync start --no-inject-changes --proxy localhost:8080 --files 'target/classes/templates' 'target/classes/static'"))
+        .addScript(scriptKey("watch:serve"), scriptCommand("browser-sync start --no-inject-changes --proxy localhost:%s --files 'target/classes/templates' 'target/classes/static'".formatted(getServerPort(properties))))
```

Comment:
There's no reason to redefine the default 8080 port in every module, it's already defined in [JHipsterServerPort](https://github.com/jhipster/jhipster-lite/blob/af7e03921f55183e544c38fd3d23718eb5b18a9b/src/main/java/tech/jhipster/lite/module/domain/properties/JHipsterServerPort.java#L6) so you can just use:
```suggestion
        .addScript(scriptKey("watch:serve"), scriptCommand("browser-sync start --no-inject-changes --proxy localhost:%s --files 'target/classes/templates' 'target/classes/static'".formatted(properties.serverPort().get())))
```
and remove the `getServerPort()` method below.

Reply:
Thanks! I was not sure if the serverPort property is generated as well when no spring-boot-mvc-empty module is added. I have removed unnecessary method